app/src/main/java/com/example/henzoshimada/feeltrip/EditMoodActivity.java

```diff
@@ -146,17 +130,20 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
             return true;
         }
     };
+    private int emojiID;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.add_edit_page);
         EditMoodActivity.context = getApplicationContext();
         inputMoodDescription = (EditText) findViewById(R.id.moodEventDescription);
+
```

Comment:
This empty space here was absolutely vital to the execution of our program. Without it, the user's phone would spontaneously explode upon launch. True story.